iree/compiler/Codegen/Common/test/convert_to_destination_passing_style.mlir

```diff
@@ -468,3 +468,62 @@ func @unused_ins_operand() {
+  %workgroup_id_y = hal.interface.workgroup.id[1] : index
+  %workgroup_count_y = hal.interface.workgroup.count[1] : index
+  %3 = affine.apply affine_map<()[s0] -> (s0 * 64)>()[%workgroup_id_y]
+  %4 = affine.apply affine_map<()[s0] -> (s0 * 64)>()[%workgroup_count_y]
+  scf.for %arg0 = %3 to %c6400 step %4 {
+    %5 = affine.apply affine_map<()[s0] -> (s0 * 64)>()[%workgroup_id_x]
+    %6 = affine.apply affine_map<()[s0] -> (s0 * 64)>()[%workgroup_count_x]
+    scf.for %arg1 = %5 to %c64 step %6 {
+      %7 = linalg.init_tensor [64, 64] : tensor<64x64xf32>
+      %8 = tensor.extract_slice %cst_0[%arg1] [64] [1] : tensor<64xf32> to tensor<64xf32>
+      %9 = flow.dispatch.tensor.load %0, offsets = [%arg0, 0], sizes = [64, 64], strides = [1, 1] : !flow.dispatch.tensor<readonly:6400x64xf32> -> tensor<64x64xf32>
+      %10 = flow.dispatch.tensor.load %1, offsets = [0, %arg1], sizes = [64, 64], strides = [1, 1] : !flow.dispatch.tensor<readonly:64x64xf32> -> tensor<64x64xf32>
+      %11 = linalg.fill(%cst_1, %7) : f32, tensor<64x64xf32> -> tensor<64x64xf32>
+      %12 = linalg.matmul {lowering.config = #iree_codegen.lowering.config<tile_sizes = [[64, 64, 0], [8, 32, 0], [0, 0, 16]], native_vector_size = []>} ins(%9, %10 : tensor<64x64xf32>, tensor<64x64xf32>) outs(%11 : tensor<64x64xf32>) -> tensor<64x64xf32>
+      %13 = linalg.generic {indexing_maps = [affine_map<(d0, d1) -> (d1)>, affine_map<(d0, d1) -> (d0, d1)>, affine_map<(d0, d1) -> (d0, d1)>], iterator_types = ["parallel", "parallel"]} ins(%8, %12 : tensor<64xf32>, tensor<64x64xf32>) outs(%7 : tensor<64x64xf32>) {
+      ^bb0(%arg2: f32, %arg3: f32, %arg4: f32):
+        %15 = arith.addf %arg2, %arg3 : f32
+        linalg.yield %15 : f32
+      } -> tensor<64x64xf32>
+      %14 = linalg.generic {indexing_maps = [affine_map<(d0, d1) -> (d0, d1)>, affine_map<(d0, d1) -> (d0, d1)>], iterator_types = ["parallel", "parallel"]} ins(%13 : tensor<64x64xf32>) outs(%7 : tensor<64x64xf32>) {
+      ^bb0(%arg2: f32, %arg3: f32):
+        %15 = arith.cmpf olt, %arg2, %cst_1 : f32
+        %16 = arith.select %15, %cst_1, %arg2 : f32
+        %17 = arith.cmpf olt, %cst, %arg2 : f32
+        %18 = arith.select %17, %cst, %16 : f32
+        linalg.yield %18 : f32
+      } -> tensor<64x64xf32>
+      flow.dispatch.tensor.store %14, %2, offsets = [%arg0, %arg1], sizes = [64, 64], strides = [1, 1] : tensor<64x64xf32> -> !flow.dispatch.tensor<writeonly:6400x64xf32>
+    }
+  }
+  return
+}
+// CHECK-LABEL: func @three_init_tensor_uses()
+//       CHECK: %[[OUTPUT:.+]] = hal.interface.binding.subspan set(0) binding(1)
+//   CHECK-NOT:   linalg.init_tensor
+//       CHECK:   %[[LOAD:.+]] = flow.dispatch.tensor.load %[[OUTPUT]]
+//   CHECK-NOT:   linalg.init_tensor
+//       CHECK:   linalg.fill(%{{.+}}, %[[LOAD]])
+//       CHECK:   linalg.generic
+//  CHECK-SAME:       outs(%[[LOAD]] :
+//       CHECK:   linalg.generic
+//  CHECK-SAME:       outs(%[[LOAD]] :
```

Comment:
Should have a new line at the end of the file.

<img width="781" alt="image" src="https://user-images.githubusercontent.com/6949191/157115597-512b8e9e-d849-4242-a65f-8c9d3e65c4c9.png">